extensions/avro-rpc/deployment/src/main/java/org/apache/camel/quarkus/component/avro/rpc/deployment/AvroRpcProcessor.java

```diff
@@ -37,7 +41,8 @@ FeatureBuildItem feature() {
     }
 
     @BuildStep
-    ReflectiveClassBuildItem registerForReflection(CombinedIndexBuildItem combinedIndex) {
+    ReflectiveClassBuildItem registerForReflection(CombinedIndexBuildItem combinedIndex,
+            BuildProducer<ReflectiveClassBuildItem> reflectiveClassProducer) {
```

Comment:
reflectiveClassProducer does not seem to be used

Reply:
I thought I removed it, so I'll do it again.

Reply:
done